src/panels/developer-tools/yaml_configuration/developer-yaml-config.ts

```diff
@@ -215,7 +215,7 @@ export class DeveloperYamlConfig extends LitElement {
         }
 
         .content {
-          padding: 28px 20px 0;
+          padding: 28px 20px 16px;
```

Comment:
Should we add margin-bottom instead?

```
margin-bottom: 24px;
margin-bottom: max(24px, env(safe-area-inset-bottom));
```

Reply:
Doesn't really make a difference, does it? So I am fine either way.

Reply:
```suggestion
          padding: 28px 20px 0;
          margin-bottom: 24px;
          margin-bottom: max(24px, env(safe-area-inset-bottom));
```